ibu-imager/cmd/postpivotcmd.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/openshift-kni/lifecycle-agent/ibu-imager/ops"
+	"github.com/openshift-kni/lifecycle-agent/ibu-imager/postpivot"
+	"github.com/openshift-kni/lifecycle-agent/internal/common"
+	"github.com/spf13/cobra"
+)
+
+// createCmd represents the create command
+var postPivotCmd = &cobra.Command{
+	Use:   "post-pivot",
+	Short: "post pivot configuration",
+	Run: func(cmd *cobra.Command, args []string) {
+		postPivot()
+	},
+}
+
+var runAsContainer bool
+
+func init() {
+
+	// Add create command
+	rootCmd.AddCommand(postPivotCmd)
+
+	// TODO: maybe better to get as os env?
+	postPivotCmd.Flags().StringVarP(&recertContainerImage, "recert-image", "e", common.DefaultRecertImage, "The full image name for the recert container tool.")
+	postPivotCmd.Flags().BoolVarP(&runAsContainer, "run-as-container", "", false, "Run post pivot from container.")
+
+}
+
+func postPivot() {
+
+	var err error
+
+	log.Printf("Post pivot operation has started")
+	var hostCommandsExecutor ops.Execute
+	if runAsContainer {
+		hostCommandsExecutor = ops.NewNsenterExecutor(log, true)
+	} else {
+		hostCommandsExecutor = ops.NewRegularExecutor(log, true)
+	}
+
+	op := ops.NewOps(log, hostCommandsExecutor)
+	postPivotRunner := postpivot.NewPostPivot(scheme, log, op, recertContainerImage,
+		"/var/lib/kubelet/config.json", common.OptOpenshift, common.KubeconfigFile)
```

Comment:
`"/var/lib/kubelet/config.json"` can be also ImageRegistryAuthFile from https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/common/consts.go#L27